internal/cluster/transition_test.go

```diff
@@ -521,14 +524,31 @@ var _ = Describe("Reset cluster", func() {
 		t := t
 		clusterId := strfmt.UUID(uuid.New().String())
 		cluster := common.Cluster{
-			Cluster: models.Cluster{ID: &clusterId, Status: swag.String(t.state)},
+			Cluster: models.Cluster{
+				ID:     &clusterId,
+				Status: swag.String(t.state),
+				Progress: &models.ClusterProgressInfo{
+					FinalizingStage:                         models.FinalizingStageWaitingForClusterOperators,
```

Comment:
Should we only set these in the test case where we're in `ClusterStatusFinalizing`?

Reply:
Agreed, let's make sure the fields match the expected stage the cluster is in